8.2.27 Release # DO NOT DELETE

su-sws-jira opened the pull request.

Commit:
STN-657:  Bug - More link in views arrow wiggle (#786)

* fix: added will-change to the more-link mixin

Adding 'will-change' to the :after and :before psuedo elements on the more-link informs the browser there will be a change to the element upon hovers with transition, eliminating browser specific rendering bugs. Such as the wiggle in Chrome for this card.

* chore: compile css

su-sws-jira committed:
8.2.27

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
Updated dependencies

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

Commit:
STN-628:  Caption/credit text styles limit (#790)

* feat:(STN-628) caption/credit text style limit

I removed the widths, the clear and the margin on the individual elements so that they can display inline with each other rather than separate cleared lines of elements. I added a small margin to the right so elements appear to have spacing when they normally wouldn't. I additionally added padding to the container to account for the margin removal from the child elements. While also adding flex and flex-end to the container so the float right elements to not appear in reverse order on the caption.

* chore: compile css

Commit:
Stn 660  caption credit refinements (#791)

* feat: use padding to adjust width for caption add auto height

Using the padding for the width of the caption allows us to keep the caption container full width for the gradient while controlling the length of the caption text.

Adding height: auto addresses hb-raised-cards bug with captions and gives it possible height decleration with any possible future utility classes that alter the cards image height.

* update: smart trim config

* chore: compile css

su-sws-jira committed:
Updated dependencies Jan 27 2021

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
restored UUID

pookmish committed:
Sparkbox Sprint 32

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
HSD8-987 Banner for imported content

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
HSD8-985 Restrict access to field default media items

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
HSD8-977 Use regex to replace new lines

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
HSD8-988 Fix the global message banner field settings

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
Reverted to older patch for config_ignore

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

sherakama committed:
BOTS-000: Block bots. (#795)

* BOTS-000: Block bots.

* original

* New patched.

* Patch cleanup.

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
block additional nuclei IP

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

pookmish committed:
block another bad ip

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

joegl committed:
HSD8-983-984: Added psychology and stanfordsciencefellows to multi-site. (#800)

* Added psychology and stanfordsciencefellows to multi-site.

Co-authored-by: joegl <joegl@users.noreply.github.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs

su-sws-jira committed:
Updated dependencies Feb 3 2021

pookmish committed:
HSD8-993 Added second department title

pookmish committed:
Added source_row_status to migrate banner lookup

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-xegqcklkqkppbtc1nncvp8w7iqvcd4qz.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90

Visual Diffs:
- https://dashboard.tugboat.qa/6008aa1bb95dbdae0818ac90/visualdiffs
